src/sentry/receivers/releases.py

```diff
@@ -78,6 +79,7 @@ def resolved_in_commit(instance, created, **kwargs):
             pass
         else:
             if repo is not None and repo.integration_id is not None:
+                issue_resolved_with_commit.send(group, sender=user_list)
```

Comment:
- i think you need to send the project here, not group? (and need to update the signal definition then)
- why is this inside the `if`?